src/main/java/com/google/sps/data/DatastoreEntityToObjectConverter.java

```diff
@@ -33,9 +33,10 @@ protected static Representative convertRepresentative(Entity entity)
         String password = (String) entity.getProperty(Constants.REP_PASSWORD);
         List<Post> posts = convertPostsFromRep(entity); 
         String intro = (String) entity.getProperty(Constants.REP_INTRO);
+        String blobKeyUrl = (String) entity.getProperty(Constants.REP_BLOB_KEY_URL);
         List<Tab> tabs = convertTabsFromRep(entity);
         long id = entity.getKey().getId();
-        return new Representative(name, title, username, password, posts, intro, tabs, id);   
+        return new Representative(name, title, username, password, posts, intro, blobKeyUrl, tabs, id);   
```

Comment:
return new Representative(
    name, title, username, password, posts, intro, blobKeyUrl, tabs, id);   

make sure the line isn't longer than 100 characters